redbot/core/commands/context.py

```diff
@@ -86,6 +86,24 @@ def __init__(self, **attrs):
         else:
             return True
 
+    async def react_quietly(
+        self, reaction: Union[discord.Emoji, discord.Reaction, discord.PartialEmoji, str]
+    ) -> bool:
+        """Adds a reaction to to the command message.
+        Returns
+        -------
+        bool
+            :code:`True` if adding the reaction succeeded.
+        """
+        try:
+            await self.message.add_reaction(reaction)
+        except (discord.HTTPException, discord.NotFound, discord.Forbidden):
+            return False
+        except discord.InvalidArgument:
```

Comment:
I'd prefer we not catch this one and allow it to automatically propagate back to cog devs (this shouldn't ever occur in a well designed cog, but I see how it can occur)